LayoutKit/Views/StackView.swift

```diff
@@ -83,6 +83,15 @@ open class StackView: UIView {
         setNeedsLayout()
     }
 
+    /**
+     Deletes all subviews from the stack.
+     */
+    open func removeArrangedSubviews() {
+        arrangedSubviews.removeAll()
```

Comment:
It looks like you forgot to actually remove the views from the view hierarchy. Please add tests to StackViewTests.swift